cloudvolume/datasource/graphene/metadata.py

```diff
@@ -402,8 +402,8 @@ def max_meshed_layer(self):
     The highest level in the ChunkedGraph that we create meshes for in this dataset.
     """
     if self.mesh_metadata and 'max_meshed_layer' in self.mesh_metadata:
```

Comment:
Looks like `and 'max_meshed_layer' in self.mesh_metadata` is not needed anymore.